app/controllers/tags_controller.rb

```diff
@@ -196,7 +196,7 @@ def create
       else
         flash[:notice] = ts('Tag was successfully created.')
```

Comment:
Thinking about it more, let's avoid making changes to existing tags altogether - wranglers should use the edit form for that and go through all the validations, instead of running some one off `update_attribute` here.

We can rewrite this section like so:
```ruby
if @tag && @tag.valid?
  if @tag.id_previously_changed?
    # Set canonical to checkbox (it's fine here because this is a new tag).
    # Flash "Tag was successfully created."
  else
    # Flash "Tag already existed and was not modified."
  end
else
  # Go back to new
end
```

Reply:
Oh, didn't know about the "id_previously_changed?" trick to differentiate between a find or a create in that case. Thanks.